### PR TITLE
[master] added lesson02

### PR DESCRIPTION
**Pull request by panser81**

### CODE REVIEW

**Comment by aperavoznikau-idt** [comment]
1. Please avoid committing the .idea folder
2. Please avoid committing .class files
3. Pel

File: Lesson02/Lesson02.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aperavoznikau-idt) [inline Lesson02/Lesson02.iml:1]: Please avoid committing

File: Lesson02/Lesson02.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+public class Lesson02{
```
Comment (aperavoznikau-idt) [inline Lesson02/Lesson02.java:1]: Use formatting (space)

File: Lesson02/Lesson02.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    public static void main(String args[]){
+        int a = 1;
+        int b = 2;
+        int c = 3;
+
+        System.out.println("Result: " + CalculateArithmeticalMean(a,b,c));
+    }
+
+    public static float CalculateArithmeticalMean(int x, int y, int z){
```
Comment (aperavoznikau-idt) [inline Lesson02/Lesson02.java:10]: For Java we are using lowercase for method naming

**Review by aperavoznikau-idt — CHANGES_REQUESTED** [review]

File: Lesson02/Lesson02.iml (hunk 1 above)
Comment (panser81) [reply Lesson02/Lesson02.iml]: excluded

File: Lesson02/Lesson02.java (hunk 2 above)
Comment (panser81) [reply Lesson02/Lesson02.java]: changed

File: Lesson02/Lesson02.java (hunk 3 above)
Comment (panser81) [reply Lesson02/Lesson02.java]: renamed

**Review by aperavoznikau-idt — APPROVED** [review]

**Pull request merged**